CHANGELOG.md

```diff
@@ -17,8 +17,12 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
 
 ## [Unreleased]
 
+### BREAKING CHANGES
+- Renamed classNames for the slots inside the `ItemLayout` component @mnajdova ([#827](https://github.com/stardust-ui/react/pull/827)) 
```

Comment:
lets do either 'class names' or '`className`s'. I would suggest the first option, as it is more correct and implementation agnostic :)